### PR TITLE
Simplify rose

### LINKED ISSUE #3: Implement conversion from GTF to the UCSC format required by rose
### Description of feature

The UCSC format required by Rose is a rather exotic file format and the developers only provide files for mouse and human. There is an nf-core module for converting GTF to genePred files available [here](https://nf-co.re/modules/ucsc_gtftogenepred). After this we should be able to follow the guide from the [issue in the ROSE repo](https://github.com/stjude/ROSE/issues/31)

### PR DESCRIPTION
**Pull request by nictru**
This PR aims on simplifiying the ROSE implementation so that fewer pipeline parameters and processes are needed.

### CODE REVIEW

**Comment by nictru** [comment]
@LeonHafner we should investigate if we can replace this with a sequence of `bedtools` commands, which would reduce the amount of code drastically

**Comment by LeonHafner** [comment]
When I run the pipeline version from your pull request, I receive fewer enhancer regions from the ROSE processes than with the current dev branch. The regions output by ChromHMM still seem to be identical to the dev branch. Can you reproduce this or do you know where this comes from?

File: modules/local/chromhmm/get_results/templates/get_results.py [hunk 1]
```diff
@@ -15,15 +15,18 @@
                   sep="\\t",
                   skiprows=1,
                   names=["chr", "start", "end", "state", "score", "strand", "start_1", "end_1", "rgb"]
-                 ).drop(columns=["strand", "score", "start_1", "end_1", "rgb"])
```
Comment (LeonHafner) [inline modules/local/chromhmm/get_results/templates/get_results.py:18]: We should update the comment for this code block as no columns are removed anymore.

**Comment by nictru** [comment]
Might be because the UCSC file that we used for ROSE is not identical with the genepred that is currently inserted into ROSE

**Comment by nictru** [comment]
We could find this out by removing the first row and column from the UCSC file and manually running the process in one of its working directories

**Review by LeonHafner — APPROVED** [review]
I tested the workflow by using the old rose ucsc file and manually overwriting the `UCSC_GTFTOGENEPRED` output.
Results are equal to `dev`, so feel free to merge!

**Comment by nictru** [comment]
Great, thanks!

**Pull request merged**